static/app/utils/gettingStartedDocs/python.spec.tsx

```diff
@@ -0,0 +1,33 @@
+// Mock the modules that cause circular dependencies
+jest.mock('sentry/utils/gettingStartedDocs/python', () => {
+  const original = jest.requireActual('sentry/utils/gettingStartedDocs/python');
+  return {
+    ...original,
+    // Mock any functions causing circular dependencies
+    getPythonProfilingOnboarding: jest.fn(),
+  };
+});
+
+import {getPythonInstallSnippet} from 'sentry/utils/gettingStartedDocs/python';
+
+describe('getPythonInstallSnippet', () => {
+  it('generates install commands with default parameters', () => {
+    const result = getPythonInstallSnippet({
+      packageName: 'sentry-sdk',
+    });
+
+    expect(result.pip).toBe(`pip install "sentry-sdk"`);
+    expect(result.uv).toBe(`uv add "sentry-sdk"`);
+    expect(result.poetry).toBe(`poetry add "sentry-sdk"`);
+  });
+
+  it('generates pip install command with minimum version and extras', () => {
```

Comment:
```suggestion
  it('generates install commands with minimum version and extras', () => {
```

now we test more than just pip :)